Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```diff
@@ -41,6 +41,11 @@
     interfaces:
         enum.AtmosAlertsComputerUiKey.Key:
             type: AtmosAlertsComputerBoundUserInterface
+        enum.WiresUiKey.Key:
+          type: WiresBoundUserInterface
+  - type: Wires
+    boardName: wires-board-name-computer
+    layoutId: ComputerAi
```

Comment:
Make a new parent like this
```
- type: entity
  parent: BaseComputer
  id: BaseComputerAiAccess
  components:
  - type: StationAiWhitelist
  - type: Wires
    boardName: wires-board-name-computer
    layoutId: ComputerAi
```
so you don't have to copy paste that part everywhere

Reply:
Makes sense, thanks